giskard/client/giskard_client.py

```diff
@@ -32,35 +32,31 @@ def __init__(self, message: str, status: int, code: str) -> None:
         self.code = code
 
 
-def explain_error(err_resp):
-    status = err_resp.get("status")
-    code = err_resp.get("message")
-    message = None
+def explain_error(status):
+    message = "Unknown error"
+    code = "error.unknown"
     if status == 401:
-        message = "Access key is invalid or expired. Please generate a new one"
-
-    if message is None:
-        if "title" in err_resp or err_resp.get("detail"):
```

Comment:
We may have to keep these.

Because some API endpoints return:
```json
{
    "title": "<error-title>",
    "message": "<error-message>",
    "detail": "<error-detail>"
}
```
in case of hub issue.

Reply:
Maybe when status code `>=500`.